model/authorizations/v1/access_review_response_type.model

```diff
@@ -19,6 +19,9 @@ struct AccessReviewResponse {
 	// Defines whether the action on the specified resource type is allowed
 	Allowed Boolean
 
+	// Defines whether an account is ocm internal
```

Comment:
missing '.' at the end of the comment (and also throughout the file)

Reply:
added